docs/app-directory/overview.md

```diff
@@ -74,6 +74,10 @@ As a vendor, you prefer for all your customers to run your latest software. Howe
 
 By hosting our own appD we can easily combine applications from various providers into one cohesive directory. Alternatively, we can connect to directories from multiple providers (in standardized format) and provide a single view over them. This reduces fragmentation in the market, allows end-users more flexibility in what apps to include in their smart desktop, and obviates the need for vendors to provide application details in diverse formats or for their customers to work out these details for themselves.
 
+## Relationship to other standards
+The App Directory's application record is similar to application manifests defined in other standards, in particular the W3C's [Web Application Manifest](https://www.w3.org/TR/appmanifest/). However, the App Directory, and by extension the application record, serve a different set of use-cases specific to application interoperability on financial services desktops, which other standards do not fully solve for. 
+
+Whereever possible, FDC3 seeks to draw inspiration from, align itself with and reference other standards - ensuring that conventions and best practices developed by those standards are reused, along with the standard itself (e.g. data formats in ISO standard formats, external links to technology specific manifest file formats etc.). For a list of standards that FDC3 references, see the [References](../references) page.
```

Comment:
@RexJaeschke @greyseer256 I'd appreciate your input on this section (and in fact this file as a whole). It was added in response to some issues raised that sought to remove this part of the standard in favour of Web Application Manifest, which didn't meet the needs of the rest of the standard. I've found this hard to word and know that it lightly duplicates some content further up this document (you can read the whole thing at https://github.com/finos/FDC3/blob/6cc60fd39fd81249aa032ecb7549495b668b8b4e/docs/app-directory/overview.md)

Your input and/or stamp of approval would be appreciated

Reply:
Kris, given my very limited knowledge of the subject matter, this looks OK.

At end of first para, consider changing "solve for" to "address".

In the second para, change "ee" to e" to avoid the obsolete spelling.